_layouts/page.html

```diff
@@ -1,14 +1,7 @@
 ---
 layout: default
 ---
-<div class="post">
 
-  <header class="post-header">
-    <h1 class="post-title">{{ page.title }}</h1>
-  </header>
-
-  <article class="post-content">
-    {{ content }}
-  </article>
-
-</div>
+    <div class='page-template {{ page.title }}'>
```

Comment:
No, I think it just ended up there as I went back in forth on trying to do it with/without templates. I've already got it moved back locally :+1: and will push up.